lib/constants/enums.ts

```diff
@@ -14,22 +14,50 @@ export enum OrderSide {
   Sell,
 }
 
-export enum Network {
+export enum XUNetwork {
+  // real coins.
   MainNet = 'mainnet',
+
+  // pre-defined testnet per currency (e.g. bitcoin: testnet3, litecoin: testnet4).
   TestNet = 'testnet',
+
+  // coins are minted by Exchange Union.
   SimNet = 'simnet',
+
+  // coins are minted privately. Seed nodes are not configurable.
   RegTest = 'regtest',
 }
 
 /**
- * Magic value indicating wire msg origin network, and used to seek to next msg when stream state is unknown
+ * LN network config per XU network.
  */
-export enum NetworkMagic {
-  MainNet = 0xd9b4bef9,
-  TestNet = 0x0709110b,
-  SimNet = 0x12141c16,
-  RegTest = 0xdab5bffa,
-}
+export const lnNetworks = {
+  [XUNetwork.MainNet]: 'mainnet',
+  [XUNetwork.TestNet]: 'testnet',
+  [XUNetwork.SimNet]: 'testnet',
+  [XUNetwork.RegTest]: 'testnet',
```

Comment:
Because on these networks we're still using `testnet` config for LND. Let me know if that's not clear or incorrect.

Reply:
I guess I'm still confused because in xud-simnet for example, we're using the `--litecoin.simnet` option and data is going in the `simnet` folder.

Reply:
I might missed it, but basically I followed the previous code, which used `testnet` as default and applied it to LND as well. There was no `simnet` support. 
Also, in LND cmd args, there's the `testnet` boolean flag only.
The `simnet` config (in our fork), is for the underlying chain. So i'm not sure which one should we follow (considering that's it's affecting only the datadir, for retrieving the macaroon file path). 

@michael1011 @offerm

Reply:
OK my memory is fuzzy on this. Simnet/regtest is unsupported in the main lnd repo for litecoin only, you're right. But I'm thinking it would make sense to target a version of lnd where simnet support does exist for litecoin (like our fork).

The previous code *would* have used the simnet/regtest folders though if that's what the network was explicitly specified as, right? In this PR we've changed the default from testnet to simnet, which I'm fine with since that's what we're using for the time being. So I think it's OK to expect someone who wants to use xud in its current state with testnet to specify the testnet network manually.

Reply:
Yes, targeting our lnd fork is necessary anyway. 

The previous code would have target the simnet/regtest folders, if specified, but it isn't specified on `xud-simnet`, and it works fine. So I figured that having `--bitcoin.simnet` config on lnd doesn't affect the folder path. But maybe it does, and it works just because lnd is being run with `--no-macaroons`.

So i’ll run a test on this, and if so, it might can remove this enum entirely.